ReactiveCocoa/Objective-C/RACCommand.h

```diff
@@ -24,7 +24,7 @@ extern NSString * const RACUnderlyingCommandErrorKey;
 
 /// A command is a signal triggered in response to some action, typically
 /// UI-related.
-@interface RACCommand : NSObject
+@interface RACCommand<__covariant InputType> : NSObject
```

Comment:
Shouldn't `InputType` be contravariant?

You've specified that `NSControl.rac_command` should be `RACCommand<__kindof NSControl *>`, but I should be able to assign a `RACCommand<NSObject *>` to that property. That command needs to be able to take an `NSControl`, but it needn't accept only `NSControl`s. Conversely, it wouldn't be right to assign a `RACCommand<NSButton *>` to it, because `NSControl` can't guarantee that an `NSButton` will be sent.

In other words, `RACCommand` inputs behave like function parameters, which are contravariant.